community-tutorials/install-prometheus-and-grafana/01-en.md

```diff
@@ -0,0 +1,167 @@
+
+1. Through the package manager:
+
+This will much likely not install the latest available version of the node-exporter but should be good enough for most setups.
+
+```
+sudo apt-get update
+sudo apt-get install -y prometheus-node-exporter
+```
+
+2. Through binary download (newest version):
+
+This way you can install any version you want, including the latest version with the latest features and bugfixes.  
+All available versions can be found on the release page: https://github.com/prometheus/node_exporter/releases
+
+```console
+wget https://github.com/prometheus/node_exporter/releases/download/v1.2.2/node_exporter-1.2.2.linux-amd64.tar.gz
+tar xf node_exporter-1.2.2.linux-amd64.tar.gz
+cd node_exporter-1.2.2.linux-amd64
+./node_exporter &
+```
+
+# Step 2 - Setup Prometheus on the "metrics" server
+
+Prometheus will pull the metrics from the node-exporter and stores the data in its database.
+
+```
+sudo apt-get install -y prometheus
+```
+
+You can now check if prometheus was successfully installed by opening the server IP followed by port 9090 in your browser.
+
+![prometheus](images/prometheus.png)
+
+
+Afterwards the node exporter targets must be configured in the prometheus server. The configuration is stored in /etc/prometheus/prometheus.yml.
+Add the following "job_name" to your "scrape_configs" and replace "ip-of-other-server" with the IP address of your second server:
+
+```yml
+scrape_configs:
+  - job_name: 'node_exporter'
+    metrics_path: "/metrics"
+    static_configs:
+      - targets:
+        - 'localhost:9100'
+        - 'ip-of-other-server:9100'
+```
+
+If you have more servers just add them one after another to the "targets" list.
+To apply the configuration changes the prometheus service needs to be restarted.
+
+```
+sudo systemctl restart prometheus.service
+```
+
+You can check the status of the service with systemctl.
+
+```
+sudo systemctl status prometheus.service
+```
+
+Prometheus now automatically scrapes node-exporter endpoints periodically on all configured targets and stores those metrics in its database.
+
+# Step 3 - Install Grafana on the "metrics" server
+
+Grafana will be used to visualize the metrics collected and stored by the prometheus service.
+
+At first you need to add the external grafana apt repository:
+
+```
+sudo apt-get install -y apt-transport-https
+sudo apt-get install -y software-properties-common wget
+wget -q -O - https://packages.grafana.com/gpg.key | sudo apt-key add -
+echo "deb https://packages.grafana.com/oss/deb stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+```
+
+Then you can install grafana itself:
+
+```
+sudo apt-get update
+sudo apt-get install -y grafana
+```
+
+You can check the status of the service with systemctl.
+
+```
+sudo systemctl status grafana.service
```

Comment:
I had to check this with 
`sudo systemctl status grafana-server.service`

Reply:
Mhm, looks like I typed (it probably got autocompleted) it correctly in my terminal but wrong in the tutorial. Thanks..
It's fixed in commit 48cc161